source/linux/How_to_Guides/Target/How_to_BeagleBadge_getting_started.rst

```diff
@@ -0,0 +1,131 @@
+===========================
+BeagleBadge Getting Started
+===========================
+
+Overview
+========
+
+The BeagleBadge is a compact development platform from BeagleBoard.org powered by the TI AM62L SoC.
+Designed for portable and low-power applications, it features built-in Wi-Fi and Bluetooth, multiple
+low power modes, and an integrated fuel gauge for precise battery power monitoring.
+
+The board provides a rich interface including an e-paper connector, DSI connector, Grove expansion,
+seven-segment displays, and an RGB LED. Fully supported in TI sources, the BeagleBadge offers flexible
+boot options (OSPI, UART, SD, USB-DFU) and runs Zephyr or Linux OS with Debian or Arago distributions,
+making it an ideal open source solution for modern IoT and HMI projects.
+
+Supported Distributions
+=======================
+
+- Debian: https://github.com/TexasInstruments/armbian-build.git branch: 2025.12-beaglebadge
+- Arago: https://github.com/TexasInstruments/meta-tisdk branch: scarthgap
+
+Low level sources
+=================
+
+- TI Linux: :file:`arch/arm64/boot/dts/til3-am62l3-beaglebadge.dts` branch: ti-linux-6.12.y
+- TI U-boot: :file:`configs/am62lx_beaglebadge_defconfig` branch ti-u-boot-2025.01
+
+Building for BeagleBadge
+========================
+
+.. code-block:: console
+
+   $ git clone https://git.ti.com/git/arago-project/oe-layersetup.git tisdk
+   $ cd tisdk
+   $ ./oe-layertool-setup.sh -f configs/arago-scarthgap-config.txt
+   $ cd build
+   $ . conf/setenv
+   $ export MACHINE=beaglebadge-ti
+   $ ARAGO_SYSVINIT=1 bitbake -k tisdk-tiny-image
+
+For more information go :ref:`here <building-the-sdk-with-yocto>`
+
+.. note::
+
+   Yocto: Due to the limited 128MB size of LPDDR on BeagleBadge, only the *tisdk-tiny-image*
+   can boot on BeagleBadge. Switching to SystemV for init system instead of SystemD will reduce
+   the memory footprint since more system services will be enabled by default with SystemD.
```

Comment:
Although this isn't necessarily true, it just happens to be true today as most services are now ported over to `systemd` (btw `systemd` is usually stylized as all lower-case) and the old `sysvinit` versions have started to bit-rot away.